src/components/axes/timeAxis.ts

```diff
@@ -445,6 +476,21 @@ export module Axis {
       return this;
     }
 
+    private _hideOverflowingTiers() {
+      var availableHeight = this.height();
+      var usedHeight = 0;
+
+      var visibleAxisTiers = this._element
+        .selectAll("." + Time.TIME_AXIS_TIER_CLASS)
+        .filter(function(d: Element, i: number) {
```

Comment:
Agree. Btw, do we have defined coding standards anywhere so I can read what is the preferred way for this codebase. Some schools prefer to define all parameters for a function because:
- Changing the function later and making use of current parameters is easier (no need to look it up)

However, I want to be consistent with the rest of the codebase. Do you think I have to remove?